README.md

```diff
@@ -87,6 +87,25 @@ Would you rather dive into the code? Then you are already in the right place!
 
 ---
 
+# Quick look at the high-level architecture
+
+![Mender architecture](mender_architecture.png)
+<!-- https://drive.google.com/file/d/1pKfJ-eRHYrDYZW7jCTHI7_D9tEF-rKDz -->
```

Comment:
@kacf is this what you've been asking for?

Reply:
Yes, perfect!